packages/blockchain-wallet-v4-frontend/src/assets/locales/index.d.ts

```diff
@@ -1551,6 +1551,13 @@ type MessagesType = {
   'scenes.home.banners.kycresubmit.resubmit': 'Resubmit Now'
   'scenes.home.banners.kycresubmit.subtitle': "We had some issues with the documents you've supplied.  Please try uploading the documents again to continue with your verification."
   'scenes.home.banners.kycresubmit.title': 'Documents Needed'
+  'scenes.home.banners.verifiedkyc.tag': 'NEW'
+  'scenes.home.banners.verifiedkyc.title': 'Keep Cash in Your Wallet'
+  'scenes.home.banners.verifiedkyc.description': 'Verify your identity to deposit cash into the Wallet and buy crypto.'
+  'scenes.home.banners.verifiedkyc.button': 'Learn More'
```

Comment:
Pretty sure there is a Learn More string somewhere already, try not to duplicate

Reply:
aha, didn't know that we reuse translations.

Reply:
FYI, we catch this at the translations level. You can have as many duplicate strings as you want, we don't get double charged. Your call on whether you want to, though.

Reply:
@amadeo-bc I thought this was manual, anyway I would like to avoid it if possible, DRYer

Reply:
It's automated. Its called 'translation memory'. Works across clients/projects. Casing inconsistencies or full stops breaks this though. Just thought i'd make you aware.